management-api-agent/src/main/java/com/datastax/mgmtapi/NodeOpsProvider.java

```diff
@@ -318,4 +321,18 @@ public void createRole(@RpcParam(name="username") String username,
     {
         return ShimLoader.instance.get().getStreamInfo();
     }
+
+    @Rpc(name = "createKeyspace")
+    public void createKeyspace(String keyspaceName, Map<String, String> replicationSettings) throws IOException
+    {
+        logger.debug("Creating keyspace {} with replication settings {}", keyspaceName, replicationSettings);
+        Map<String, Integer> settings = new HashMap<>();
```

Comment:
[minor] using map(transforming) would stop us from creating new local map with new
		
Map<String, Integer> settings = replicationSettings.entrySet().stream()
				.collect(
						Collectors.toMap(Map.Entry::getKey, entry ->Integer.valueOf(entry.getValue()))
				);

Reply:
I was only checking something with this code, but yes, the replication settings will usually always be a pair of String, Integer